fedimint-client/src/module/mod.rs

```diff
@@ -772,6 +781,11 @@ pub trait IClientModule: Debug {
     async fn handle_cli_command(&self, args: &[ffi::OsString])
         -> anyhow::Result<serde_json::Value>;
 
+    async fn handle_rpc(
+        &self,
```

Comment:
Where's the rpc name? Seems to me that name + payload is a more pragmatic approach, no?